### DISCUSSION TITLE
Docs: How to handle "too many options, please keep on typing"?

### DISCUSSION THREAD

**Discussion by guettli** [orchidjs/tom-select#134] [Q&A] [answered]
I look at the docs: https://tom-select.js.org/docs/

And I read:

> load(query, callback) | Invoked when new options should be loaded from the server. Called with the current query string and a callback function to call with the results when they are loaded (or nothing when an error arises).

I have several thousand options in the DB. If the user just types "e", then there are far too many options.

How to tell the user "too many options, please keep on typing"?

Please explain this in the docs.

Thank you very much.

**Comment by oyejorge** [comment, marked as answer]
Take a look at the shouldLoad() callback and not_loading template, they should cover what you're looking for
